lib/src/hash_sink.dart

```diff
@@ -25,10 +26,6 @@ abstract class HashSink implements Sink<List<int>> {
   /// used across invocations of [_iterate].
   final Uint32List _currentChunk;
 
-  /// Messages with more than 2^64-1 bits are not supported.
-  /// So the maximum length in bytes is (2^64-1)/8.
-  static const _maxMessageLengthInBytes = 0x1fffffffffffffff;
```

Comment:
Use 0x0003FFFFFFFFFFFF instead
The largest number of bytes for which we can count the bits portably across VM and JavaScript